cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -115,7 +116,7 @@ def pytest_generate_tests(metafunc):
                 (machine_id, depot_type)
 
             hostname = depot_type_content["hostname"]
-            if depot_type == "ftp" and machine_id not in machines_ftp:
+            if depot_type in ["ftp", "anon_ftp"] and machine_id not in machines_ftp:
```

Comment:
We shouldn't be doing this. We add machines to `machines_ftp` because we later use the data to get an ftp client that is used to verify results of the `ftp`, `nfs` and `samba` tests. In case of `anon_ftp`, we don't actually need to save this data. If this was added to make sure that the assert on line 132 doesn't fail, its incorrect. What we should actually do is run the assert on line 132 and fetch the ftp credentials from `machines_ftp` only if the depot_type is not `anon_ftp`.